File: Java/src/com/jcuadros/dbconn/MyConnection.java
package com.jcuadros.dbconn;

import java.sql.Connection;
import com.mysql.cj.jdbc.Driver;
import java.sql.DriverManager;
import java.sql.SQLException;

public class MyConnection {
	public static final String URL = "jdbc:mysql://localhost:3306/sakila?useSSL=false";
	public static final String USER = "root";
	public static final String PASS = "root";

	public static Connection getConnection() {
		try {
			DriverManager.registerDriver(new Driver());
			return DriverManager.getConnection(URL, USER, PASS);
		} catch (SQLException ex) {
			throw new RuntimeException("Error connecting to the database", ex);
		}
	}
	
}

File: Scala/CSC450 Research/src/com/jcuadros/prj/MyConnection.scala
import java.sql.Connection
import java.sql.DriverManager

object MyConnection {

  def main(args: Array[String]): Unit = {

    var connection: Connection = null

    try {
      Class.forName("com.mysql.cj.jdbc.Driver")
      connection = DriverManager.getConnection("jdbc:mysql://localhost:3306/sakila?useSSL=false", "root", "root")

      val statement = connection.createStatement()

      var index = 0
      var maxID = 330266

      for (index <- 1 until 100000) {
        statement.execute("INSERT INTO researchResults(`id`, `language`, `iteration`, `startTime`) VALUES ( " + maxID + ", 'Scala', " + index + ", Now(6))")
        statement.executeQuery("SELECT * FROM sakila.seniorresearch LIMIT " + index)
        statement.executeUpdate("UPDATE researchResults SET `endTime` = Now(6) WHERE id = " + maxID)
        maxID = maxID + 1
      }

    } catch {
      case e => e.printStackTrace()
    }
    connection.close()
  }
}